### ISSUE TITLE
how is generated xterm/index.html?

### ISSUE THREAD

**Issue by whren** [elebeaup/live-coding-server#3]
From my understanding `xterm/index.html` is generated following [https://github.com/tsl0922/ttyd/tree/master/html#publish](https://github.com/tsl0922/ttyd/tree/master/html#publish) with `yarn run build.`
As stated `Run yarn run build, this will compile the inlined html to ../src/html.h.`

Is this correct?
In this case I think it should be documented in the README, in order to be able to update ttyd xterm management.

**Comment by elebeaup** [comment]
Yes, that's it. The index.html comes from tsl0922/ttyd but it is an old version. I haven't tested the latest version.
Another solution to get started is to go to the demo folder and learn from samples. There is an example to share web terminal with a web component.
The documentation is still very sparse.
